### DISCUSSION TITLE
Flex not work like expect...

### DISCUSSION THREAD

**Discussion by IRediTOTO** [tailwindlabs/tailwindcss#2780] [Help]
[https://play.tailwindcss.com/pDeoIfrfAl?size=540x720](https://play.tailwindcss.com/6HTvN1qe7G?size=540x720)
Here is demo, can you tell me why the flex not overflow-x scroll... I don't understand what did I do wrong... I should be 8 colums of img in desktop and scroll x on mobile...
Thank you

**Comment by erhankaradeniz** [comment]
You need to set `flex-shrink-0` on the div elements, see: [https://play.tailwindcss.com/Q8YMjX7zX6?size=540x720](https://play.tailwindcss.com/Q8YMjX7zX6?size=540x720)

**Reply by IRediTOTO** [reply]
Thank you so much :| I usually use flex, after try many times, i fixed it but I can't understand why.

**Reply by erhankaradeniz** [reply]
You have to set the shrink-0 because flex will try to fit all elements it has, by setting flex-shrink-0 you force the elements to stay at the defined width.
